### ISSUE TITLE
install target fails on Mint 17.3

### ISSUE THREAD

**Issue by caryan** [BBN-Q/libaps2#49]
@blakejohnson after bda0cbbb85e59799543d18abacfaf19763e678da `make install` now fails with:

```shell
-- Installing: /home/cryan/Programming/Repos/libaps2/build/install/lib/libaps2.so.0.6.0
-- Installing: /home/cryan/Programming/Repos/libaps2/build/install/lib/libaps2.so.0
-- Installing: /home/cryan/Programming/Repos/libaps2/build/install/lib/libaps2.so
-- Set runtime path of "/home/cryan/Programming/Repos/libaps2/build/install/lib/libaps2.so.0.6.0" to "$ORIGIN"
-- Installing: /home/cryan/Programming/Repos/libaps2/build/install/include/libaps2.h
CMake Error at cmake_install.cmake:256 (FILE):
  file INSTALL cannot find "/usr/bin/../lib64/libgcc_s.so.1".
```

It seems [Ubuntu derivatives put](http://packages.ubuntu.com/trusty/amd64/libgcc1/filelist
) `libgcc` in `/lib/x86_64-linux-gnu/`

**Comment by caryan** [comment]
@blakejohnson could you give the [fix/cmake-library-finding](https://github.com/BBN-Q/libaps2/tree/fix/cmake-library-finding)  a try on your Centos deployment box.  Because it is expected the appropriate `libgcc` and `libstdc++` will be linked by the chosen compiler we have to specify their full name with the `lib` prefix and version suffix.

**Comment by caryan** [comment]
Actually 0024919 doesn't work for HDF5 as it just copies broken symlinks. It might be possible to use `get_filename_component` with `REALPATH` to resolve the symlink

**Comment by blakejohnson** [comment]
Fixed in 1527fc597b0dc0f8d1f7e668c5a1910083bffea7.
